nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md

```diff
@@ -242,7 +241,7 @@ Use to set a top-limit for the default CPU requirement for each process.
 Should be a string in the format integer-unit. eg. `--max_cpus 1`
 
 ### `--plaintext_email`
-Set to receive plain-text e-mails instead of HTML formatted.
+Set to receive plain-text e-mails instead of HTML formatted.  
 
-### `--multiqc_config`
+### `--multiqc_config`
```

Comment:
Don't see a change here, probably different encoding?

Reply:
Im a bit puzzled too.

The `--multiqc_config` section isnt rendered properly in the current `usage.md`:
https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/usage.md#--plaintext_email

Im not sure whether the whitespace in the comment above sorts it out but its correctly rendered in this version:
https://github.com/drpatelh/tools/blob/dev/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/usage.md#--multiqc_config

I got this working for the docs in `nf-core/atacseq` so I copied it across and looked at the diff. Weird but it seems to work. I was staring at it for ages!

Reply:
It's a non breaking space that can be achieved by holding shift and pressing space*. It looks identical in text editors and GitHub, but breaks GitHub markdown processing. Delete and press space again carefully and everything will work as expected.

_* Learnt from a frustratingly large number of times where I've done this myself_

Reply:
holy, @ewels wtf. Yeah next guess would have been to check character encoding ...

Reply:
Seeing as what i have done is doing the trick is it ok to leave it as it is? Not sure if it's likely to break something else.

Reply:
Should be fixed now in latest commit. I think...